src/src/pages/capabilities/details.js

```diff
@@ -45,6 +47,18 @@ function CapabilityDetailsPageContent() {
 
   const pagetitle = isDeleted ? `${name} [Deleted]` : name;
 
+  const [showJsonMetadata, setShowJsonMetadata] = useState(false);
+  useEffect(() => {
+    if (
+      links &&
+      links.metadata &&
+      links.metadata.allow.includes("PUT") &&
+      links.metadata.allow.includes("GET")
```

Comment:
Should we not show JSON metadata if people are allowed to read (i.e. `GET`) it?
Even if they cannot update or write (i.e. `PUT`) it?

Reply:
Something we can add later when we know how we want to use it